src/pages/Scan.vue

```diff
@@ -85,10 +85,13 @@ export default {
                         // Add error message for invalid and fail codes.
                         if (
                             response.data.invalid.length > 0
-                            || response.data.fail.length > 0
                         ) {
                             this.showFail();
-                            this.errorMessage = "Please enter a valid code.";
+                            this.errorMessage = "The voucher code you entered is not valid. Please try again.";
+                        } else if (
+                            response.data.fail.length > 0
+                        ) {
+                            this.errorMessage = "The voucher code has you entered has previously been submitted. Please try again.";
                         } else {
```

Comment:
done. And added info to cards... 

https://trello.com/c/o9kY8iNK/332-add-message-component-to-display-success-and-error-messages 

https://trello.com/c/o9kY8iNK/332-add-message-component-to-display-success-and-error-messages